src/App.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { Button } from './components/Button/Button';
+
+function App() {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const getData = async () => {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const data = await response.json();
+
+    getAllAvatars(data);
+  };
+
+  const getAllAvatars = async (data) => {
+    let newUsersList = [];
+
+    for (let i = 0; i < data.length; i++) {
+      const response = await fetch(
+        `https://avatars.dicebear.com/api/male/${data[i].username}.svg`
+      );
+      const avatar = await response.text();
+      newUsersList.push({ ...data[i], avatar });
+    }
+
+    setUsers(newUsersList);
+  };
+
+  return (
+    <div className='App'>
+      <Button onClick={getData} text='Obtener usuarios' />
+
+      {users.length === 0 ? (
```

Comment:
this is unnecessary in javascript just asking if the length is the same and better than asking if identical to 0.

**Try to don't do conditional rendering, is a BAD practice on REACT**

```suggestion
      {!users.length ? (
```